openmdao/vectors/default_vector.py

```diff
@@ -426,6 +426,26 @@ def set_const(self, val):
         for data in itervalues(self._data):
             data[:] = val
 
+    def dot(self, vec):
+        """
+        Compute the dot product of the real parts of the current vec and the incoming vec.
+
+        Parameters
+        ----------
+        vec : <Vector>
+            the vector whose values self is set to.
```

Comment:
It looks like this comment may have been copied from `set_vec` without being updated.